gtsam/geometry/Line3.h

```diff
@@ -146,7 +146,30 @@ class Line3 {
  */
 Line3 transformTo(const Pose3 &wTc, const Line3 &wL,
                   OptionalJacobian<4, 6> Dpose = boost::none,
-                  OptionalJacobian<4, 4> Dline = boost::none);
+                  OptionalJacobian<4, 4> Dline = boost::none) {
```

Comment:
I had trouble with definitions in headers. Should be inline? But I don't understand why we can't just export here and leave definition in .cpp?